src/pages/callouts/[id]/gallery.vue

```diff
@@ -54,6 +54,11 @@ meta:
       @close="router.push({ hash: '' })"
       @click.stop
     />
+    <CalloutIntroPanel
+      :callout="callout"
+      :show="!!introOpen"
+      @close="handleCloseIntro"
```

Comment:
I would just inline `introOpen = false` here, it's probably clearer than having a method for such a simple thing